### PR TITLE
Kaydunov_EH1T1

### PR DESCRIPTION
**Pull request by GardDen**

### CODE REVIEW

**Review by AlexMackevich — DISMISSED** [review]

**Review by gornovanton1992 — CHANGES_REQUESTED** [review]
please, finish task

**Review by gornovanton1992 — APPROVED** [review]

**Review by AlenaPalasionak — APPROVED** [review]
lgfm

**Comment by GardDen** [comment]
> please, finish task

Finished.

**Comment by GardDen** [comment]
> lgfm

Алена, ты просмотрела реквест но не апрувнула.

**Review by NikaLu — COMMENTED** [review]
Need to correct

**Comment by GardDen** [comment]
> Need to correct
What I must to correct?

**Review by VasiliZas — APPROVED** [review]

File: homework/english/kaydunov/lecture1/task1.md [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+2 How old are you? - I am thirty old.
+3 Where do you live? - I am from Belarus, Gomel. I live in a private house.
+4 What do you do? - I am a programmer.
+5 What do you like doing in your free time? - I like latina dance. I playing the football.
+6 Would you like to be a programmer? Why? - I like to be a programmer because it is cool, is 
+constant self-study, is automation of routine, is blockchein, is robots, 
+augmented reality, is project "Venera", is "Ilon Mask". Yes, I heard, what Java using in 
+space rocket. Programing is future.
+7 Did you learn English before? Would you like to improve your English? What is the hardest thing about learning English?
+Yes, I learn English before. I wont improve my speaking English. The hardest thing about learning English is speaking English.
```
Comment (NikaLu) [inline homework/english/kaydunov/lecture1/task1.md:190]: I want to improve my spoken language.

**Review by NikaLu — COMMENTED** [review]
> This is first lesson english.

> My home work will there soon.

**Review by AnzhelaKhamitskaya — DISMISSED** [review]

File: homework/english/kaydunov/lecture1/task1.md [hunk 2]
```diff
@@ -0,0 +1,191 @@
+# [![Generic badge](https://img.shields.io/badge/homework-ENGLISH-0008fa.svg)](https://moodle.jrr.by/)
+#EH1
+##englich.homework1
+
+This is first lesson english.
```
[the remark this answers is not in the document]
Comment (GardDen) [reply homework/english/kaydunov/lecture1/task1.md:5]: - [ ]

File: homework/english/kaydunov/lecture1/task1.md [hunk 3]
```diff
@@ -0,0 +1,191 @@
...
+My home work will there soon.
```
[the remark this answers is not in the document]
Comment (GardDen) [reply homework/english/kaydunov/lecture1/task1.md:6]: +

**Review by AnzhelaKhamitskaya — APPROVED** [review]

**Review by NikaLu — APPROVED** [review]

**Review by shpaser2 — APPROVED** [review]
Lgfm

**Pull request merged**